Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/AbilData.xml

```diff
@@ -7938,6 +7938,45 @@
         <TargetFilters value="Ground,Visible;Self,Structure,Heroic,Missile,Destructible,Stasis,Dead,Hidden,Invulnerable"/>
         <CmdButtonArray index="Execute" DefaultButtonFace="AP_HercGrapple" Requirements="AP_HaveHERCGrapplePull"/>
     </CAbilEffectTarget>
+    <CAbilArmMagazine id="AP_BomberHanger">

```

Comment:
Note the correct spelling is "hangar". A typo like this can be fine, except the very next abil spells it correctly, and the consistency is definitely more important (for searching purposes)